src/Microsoft.IdentityModel.Tokens/SecurityTokenDescriptor.cs

```diff
@@ -65,6 +71,11 @@ public class SecurityTokenDescriptor
         /// </summary>
         public DateTime? NotBefore { get; set; }
 
+        /// <summary>
+        /// Gets or sets the <see cref="Dictionary{TKey, TValue}"/> which represents the JWT token payload.
+        /// </summary>
+        public Dictionary<string, object> Payload { get; set; }
```

Comment:
Setting the type of the Payload property to be an IDictionary<string, object> would help to decouple the contract of the SecurityTokenDescriptor type from the underlying implementation.

Reply:
@CBaud 	Thanks for chiming in. Yes IDictionary...

Reply:
Yes, this makes sense.